CI/before_install/linux_qt6.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 
-sudo apt remove needrestart
 sudo apt-get update
 
 # Dependencies
+# In case of change in dependencies list please also update:
+# - developer docs at docs/developer/Building_Linux.md
```

Comment:
This place should actually be easier to find.
Since if you were to add new library CI will fail and you'll have to update this file and (hopefully) see this message.

Can mention it to docs for completeness sake.